chain/cosmos/module_gov.go

```diff
@@ -0,0 +1,210 @@
+package cosmos
```

Comment:
these functions were moved from chain_node.go. Only `GovQuery*` functions were added. 

YOu can now specify to query govv1beta1 (< v47) or govv1 (SDK v50)